src/cpp/rediscluster.cpp

```diff
@@ -179,7 +183,7 @@ CommandReply RedisCluster::rename_tensor(const std::string& key,
     uint16_t new_key_hash_slot = this->_get_hash_slot(new_key);
 
     if(key_hash_slot == new_key_hash_slot) {
-        Command cmd;
+        MultiKeyCommand cmd;
```

Comment:
``rename_tensor`` is actually a ``CompoundCommand`` composed of a get and set for rediscluster.